src/chapters/configuring.xml

```diff
@@ -6195,6 +6195,33 @@ guaclog: INFO: All files interpreted successfully.</computeroutput>
                 </section>
             </section>
         </section>
+        <section xml:id="parameter-prompting">
+            <title>Parameter prompting</title>
+            <para><indexterm>
+                <primary>parameters</primary>
+                <secondary>prompt</secondary>
+            </indexterm>In certain situations Guacamole may determine that additional
+                information is required in order to successfully open or continue a
+                connection. In these scenarios guacd will send an instruction back to
+                the client to retrieve that information, which will result in the user
+                being prompted for those additional parameters.</para>
+            <para>Currently the only parameters that will trigger this prompt to the
+                user are authentication requests for the RDP and VNC protocols where
+                authenticators were not provided as part of the connection configuration.
+            </para>
+            <important>
+                <para>It is important to note that requests for parameters will only be
+                    generated in the case where that information has not already been
+                    provided as part of the connection. The user will never be asked for
+                    parameters that replace or override connection parameters where
+                    values have been configured as part of the connection, including
+                    authentication information. For example, if a connection to a
+                    RDP server specifies a username and password, and that username
```

Comment:
I think this should be clarified to refer to a "connection configuration" or "the configuration of a connection" or similar, rather than just "connection".